src/AggregateFunctions/AggregateFunctionWindowFunnel.h

```diff
@@ -226,18 +226,20 @@ class AggregateFunctionWindowFunnel final
         events_size = arguments.size() - 1;
         window = params.at(0).safeGet<UInt64>();
 
-        strict = false;
+        strict_deduplication = false;
         strict_order = false;
         strict_increase = false;
         for (size_t i = 1; i < params.size(); ++i)
         {
             String option = params.at(i).safeGet<String>();
-            if (option == "strict")
-                strict = true;
+            if (option == "strict_deduplication")
+                strict_deduplication = true;
             else if (option == "strict_order")
                 strict_order = true;
             else if (option == "strict_increase")
                 strict_increase = true;
+            else if (option == "strict")
+                throw Exception{"strict is replaced with strict_deduplication in Aggregate function " + getName(), ErrorCodes::BAD_ARGUMENTS};
```

Comment:
This is a good solution until documentation in other languages will not be fixed.